.circleci/config.yml

```diff
@@ -0,0 +1,323 @@
+              echo 'export PATH=$PATH:`yarn bin`' >> $BASH_ENV
+              echo 'export PATH=$PATH:$ANDROID_HOME/tools/gradle/bin' >> $BASH_ENV
+      - run:
+          name: Install Gradle 4.10.2
+          command: |
+            wget -O /tmp/gradle.zip https://downloads.gradle.org/distributions/gradle-4.10.2-bin.zip
+            unzip -d $ANDROID_HOME/tools /tmp/gradle.zip
+            mv $ANDROID_HOME/tools/gradle-4.10.2 $ANDROID_HOME/tools/gradle
+      - run:
+          name: Download Keystore File
+          command: yarn run azure-filestore download -f $KEYSTORE
+          working_directory: cordova
+      - run:
+          name: Download Google Play Store Key file
+          command: |
+           yarn run azure-filestore download -d google-play -f $GOOGLE_PLAY_KEY_FILE
+           mv $GOOGLE_PLAY_KEY_FILE ~/code/fastlane
+          working_directory: ~/code
+      - run:
+          name: android build
+          command: bundle exec rake ${ENVIRONMENT} android app:build
+          working_directory: ~/code/cordova
+      - store_artifacts:
+          path: cordova/platforms/android/build/outputs/apk/
+      - run:
+          name: release android build
+          command: bundle exec fastlane android ${ENVIRONMENT}
+          working_directory: ~/code
+
+  ios_build_and_deploy:
+    macos:
+      xcode: "9.2.0"
+    working_directory: ~/code
+    shell: /bin/bash --login -eo pipefail
+    environment:
+        EMBER_CLI_CORDOVA: "1"
+        CIRCLE_ARTIFACTS: /Users/distiller/code/cordova/platforms/ios/build/device
+
+    steps:
+      - checkout
+      - run: *yarn
+      - save-cache: *save-yarn-cache
+      - run: yarn run bower install
+      - save-cache: *save-bower-cache
+      - run:
+          name: Setup Environment variables
+          command: |
+            echo 'export PATH=$PATH:`yarn bin`' >> $BASH_ENV
+            echo 'export ENVIRONMENT=$(if [ "$CIRCLE_BRANCH" == "live" ]; then echo production; else echo staging; fi)' >> $BASH_ENV
+            echo 'export PROVISIONING_PROFILE=$(if [ "$CIRCLE_BRANCH" == "live" ]; then echo $ENCODED_PROD_PROVISIONING_PROFILE; else echo $ENCODED_STAGING_PROVISIONING_PROFILE; fi)' >> $BASH_ENV
+            echo 'export CERTIFICATES=$(if [ "$CIRCLE_BRANCH" == "live" ]; then echo $ENCODED_PROD_CERT; else echo $ENCODED_STAGING_CERT; fi)' >> $BASH_ENV
+            echo 'export STAGING=$(if [ "$CIRCLE_BRANCH" == "live" ]; then echo false; else echo true; fi)' >> $BASH_ENV
+      - run:
+          command: |
+              npm rebuild node-sass
+              echo "Building for staging (true/false): $STAGING"
+              yarn run ember build --environment=production
+      - run: mv dist/ cordova/www
+      - persist_to_workspace:
+          root: .
+          paths:
+            - cordova/www
+      - run: bundle check || bundle install --path vendor/bundle
+      - save-cache: *save-bundler-cache
```

Comment:
Need to restore bundler cache here too. Perhaps build an ios specific one in case any gems have pre-built binaries.